### PR TITLE
add servo dual target

### PR DESCRIPTION
**Pull request by jamesadevine**

### CODE REVIEW

File: targets/_example/profile/module.c [hunk 1]
```diff
@@ -19,7 +19,7 @@ const servo_params_t servo_params = {
 void app_init_services() {
     // see jacdac-c/services/jd_services.h for the services that can be
     // enabled here you can enable zero or more services
-    servo_init(&servo_params);
+    servo_init(&servo_params, 0xff);
```
Comment (mmoskal) [inline targets/_example/profile/module.c:22]: I usually just pass -1

**Review by mmoskal — APPROVED** [review]

File: targets/_example/profile/module.c (hunk 1 above)
Comment (jamesadevine) [reply targets/_example/profile/module.c:22]: Would you like me to change the parameter to expect -1?

Comment (mmoskal) [reply targets/_example/profile/module.c:22]: well, if it takes uint8_t then -1 and 0xff are the same

Comment (mmoskal) [reply targets/_example/profile/module.c:22]: maybe we should defined PIN_NC or something to be -1

Comment (mmoskal) [reply targets/_example/profile/module.c:22]: or maybe you're right, #define NO_PIN 0xff will be probably less error prone

Comment (jamesadevine) [reply targets/_example/profile/module.c:22]: okie dokie!

**Pull request merged**